docs/UserGuide.md

```diff
@@ -14,19 +14,19 @@ GREWZ is a **desktop app for managing student details, optimized for use via a C
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `GREWZ.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `GREWZ.jar` from [here](https://github.com/AY2223S1-CS2103T-W12-4/tp/releases).
```

Comment:
Do we need to change the number for the ordered list?

Reply:
oh yes